bin/build-plugin-zip.sh

```diff
@@ -99,8 +101,7 @@ zip -r gutenberg.zip \
 	blocks/library/*/*.php \
 	post-content.js \
 	$vendor_scripts \
-	{blocks,components,date,editor,element,hooks,i18n,data,utils,edit-post,viewport}/build/*.{js,map} \
```

Comment:
I don't believe that `*.{js,map}` bit was up-to-date. In any case, we don't expect `.map` files in the ZIP file, to my knowledge. Hence my removal.